addons/web/static/src/xml/crash_manager.xml

```diff
@@ -2,7 +2,7 @@
 <templates id="template" xml:space="preserve">
 
 <div t-name="CrashManager.warning" class="o_dialog_warning" role="alert">
-    <pre t-esc="widget.message"/>
```

Comment:
Sure the messages cannot contain line feeds ?

Reply:
Yes there was a bug report here: https://www.odoo.com/web#view_type=form&model=project.task&id=2042028&active_id=2042028&menu_id=
The pre tag was not there before so I removed it.

Reply:
As discussed: change in <p/> t-esc, white-space: pre-wrap so that it works like before... we'll wait for JKE's return to see if the case with HTML image really exists.